tests/cases/widgets/slider_basic.slint

```diff
@@ -0,0 +1,50 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+
+import { Slider } from "std-widgets.slint";
+export component TestCase inherits Window {
+    width: 100px;
+    height: 100px;
+    slider := Slider {
+        minimum: 0;
+        maximum: 100;
+        value: 10;
+    }
+
+    forward-focus: slider;
+    out property <bool> slider-focused<=> slider.has_focus;
+    callback released<=>slider.released;
+}
+
+/*
+
+
+```rust
+use slint::platform::Key;
+use std::cell::RefCell;
+use std::rc::Rc;
+
+let instance = TestCase::new().unwrap();
+slint_testing::send_mouse_click(&instance, 5., 5.);
```

Comment:
should that mouse click be sent after we set the on_released callback so we check that the callback is emit (or not) when the mouse is released.

(would be nice to also test the changed signal at the same time)